doctrine/reverse_engineering.rst

```diff
@@ -4,6 +4,14 @@
 How to Generate Entities from an Existing Database
 ==================================================
 
+.. caution::
```

Comment:
I guess that this was the wrong branch when you did a commit?

Reply:
I have no clue what happened haha, that's why I made a new PR.